### PR TITLE
SEO Improvements

### PR DESCRIPTION
**Pull request by thostetler**
Adds a couple plugins to improve SEO

`jekyll-seo-tag`: to generate JSON-LD and meta tags for each page
`jekyll-last-modified-at`: to generate last-modified meta tags for each page

Switched to using a Gemfile for package management.  This isn't necessary, just nice so you don't have to reinstall by hand.

A couple of the .scss files were giving me build errors, fixed a small issue in there, and reformatted some.

image

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -3,8 +3,16 @@ title: ADS news, blogs, and help pages
 url: https://ui.adsabs.harvard.edu
 email: adshelp@cfa.harvard.edu
 description: All your questions will be answered here.
+
+# Social
 twitter_username: adsabs
 github_username: adsabs
```
Comment (kelockhart) [inline _config.yml:9]: Can we remove these two lines, since they're below now, or are both sets needed?

**Review by kelockhart — COMMENTED** [review]
One small comment from me, but I'll let @mugdhapolimera validate that the last modified date works for her needs.

**Review by mugdhapolimera — APPROVED** [review]
Everything looks good and renders well locally!

File: _config.yml (hunk 1 above)
Comment (thostetler) [reply _config.yml]: Yep agreed, just checked and they aren't being referenced in the source code, so I think safe to remove.

**Pull request merged**